### DISCUSSION TITLE
編集リクエストまわりの動線

### DISCUSSION THREAD

**Discussion by scivola** [increments/qiita-discussions#148] [要望・アイデア・不具合]
編集リクエストまわりの動線に関して使いづらく感じています。

編集リクエストのうち，誤字や用語の訂正は，他にも同じことを考える人がいるものです。
そこで，編集リクエストを作る前に，既に同様のリクエストが無いかどうか確認することになります。

ところが，「編集リクエスト一覧」のページからは編集リクエストが作れません。いったん記事に戻る必要があります。
レスポンスが必ずしも早くないこともあってストレスを感じます。
ついでに言うと，この画面から記事に戻るリンクは，慣れるまで分かりづらく感じました。記事タイトルの冒頭だけだからです。

また編集リクエストを作る画面で，「そうだ，既にリクエストがあったりしないかな」と思っても，やはりいったん記事に戻る必要があります。

**Comment by kskwtnk** [comment]
@scivola 
ご意見ありがとうございます！

今ちょうど編集リクエストページを改修していまして、そのリリースでこちらの内容も盛り込めないか調整してみます！

**Comment by kyntk** [comment]
@scivola 
記事の編集リクエストページをリニューアルしまして、ここに編集リクエスト作成への導線を追加しました！
https://qiita.com/release-notes#2022-03-22
